### PR TITLE
NO-ISSUE - Human dmesg timestamps

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

**Review by osherdp — APPROVED** [review]
/lgtm

File: src/logs_sender/send_logs.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func getMountLogs(l LogsSender, outputFilePath string) error {
 
 func getDmesgLogs(l LogsSender, outputFilePath string) error {
 	log.Infof("Running dmesg")
-	stderr, exitCode := l.ExecuteOutputToFile(outputFilePath, "dmesg")
+	stderr, exitCode := l.ExecuteOutputToFile(outputFilePath, "dmesg", "-T")
```
Comment (romfreiman) [inline src/logs_sender/send_logs.go:171]: basics :)

**Pull request merged**